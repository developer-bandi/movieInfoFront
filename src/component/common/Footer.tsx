import styled from 'styled-components';

const Footer = () => {
  return (
    <FooterBlock>
      <Content>이 사이트는 개인 포트폴리오용 사이트입니다</Content>
      <Content>Copyright 2022. 김상두 all right reserved</Content>
      <Content>Source by TMDB</Content>
    </FooterBlock>
  );
};

const FooterBlock = styled.footer`
  width: 100%;
  height: 5rem;
  background: #425a73;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  padding-top: 1rem;
  padding-bottom: 1rem;
  @media screen and (max-width: 500px) {
    font-size: 8px;
    height: 1.5rem;
  }
`;

const Content = styled.div`
  color: white;
  & + & {
    margin-top: 0.5rem;
  }
`;

export default Footer;
