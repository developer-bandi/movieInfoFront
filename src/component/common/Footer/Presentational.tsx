import styles from './Style';

const Footer = () => {
  return (
    <styles.MainBlock>
      <styles.Content>
        이 사이트는 개인 포트폴리오용 사이트입니다
      </styles.Content>
      <styles.Content>Copyright 2022. 김상두 all right reserved</styles.Content>
      <styles.Content>Source by TMDB</styles.Content>
    </styles.MainBlock>
  );
};

export default Footer;
