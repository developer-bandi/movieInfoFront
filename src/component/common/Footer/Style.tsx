import styled from 'styled-components';

const MainBlock = styled.footer`
  width: 100%;
  height: 5rem;
  background: #425a73;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  padding-top: 1rem;
  padding-bottom: 1rem;
  @media screen and (max-width: 500px) {
    font-size: 8px;
    height: 1.5rem;
  }
`;

const Content = styled.div`
  color: white;
  & + & {
    margin-top: 0.5rem;
  }
`;

const styles = { MainBlock, Content };

export default styles;
