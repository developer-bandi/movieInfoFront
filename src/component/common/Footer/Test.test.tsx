import { render } from '@testing-library/react';
import Footer from './Presentational';

describe('<Footer/>', () => {
  it('전체 화면 검증', () => {
    const utils = render(<Footer />);
    expect(utils.container).toMatchSnapshot();
  });
});
