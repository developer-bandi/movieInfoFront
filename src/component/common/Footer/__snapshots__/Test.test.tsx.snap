// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Footer 스냅샷 비교 1`] = `
<div>
  <footer
    class="sc-bdvvtL iRKeFJ"
  >
    <div
      class="sc-gsDKAQ kXergA"
    >
      이 사이트는 개인 포트폴리오용 사이트입니다
    </div>
    <div
      class="sc-gsDKAQ kXergA"
    >
      Copyright 2022. 김상두 all right reserved
    </div>
    <div
      class="sc-gsDKAQ kXergA"
    >
      Source by TMDB
    </div>
  </footer>
</div>
`;

exports[`<Footer/> 전체 화면 검증 1`] = `
<div>
  <footer
    class="sc-bdvvtL iRKeFJ"
  >
    <div
      class="sc-gsDKAQ kXergA"
    >
      이 사이트는 개인 포트폴리오용 사이트입니다
    </div>
    <div
      class="sc-gsDKAQ kXergA"
    >
      Copyright 2022. 김상두 all right reserved
    </div>
    <div
      class="sc-gsDKAQ kXergA"
    >
      Source by TMDB
    </div>
  </footer>
</div>
`;
